R/get_rcode.R

```diff
@@ -62,6 +62,14 @@ get_rcode <- function(datasets = NULL,
                       title = NULL,
                       description = NULL) {
   checkmate::assert_class(datasets, "FilteredData", null.ok = TRUE)
+
+  lifecycle::deprecate_warn(
+    when = "0.11.2",
```

Comment:
is this right? Also check others in this PR

Reply:
I've updated this version here